lib/badges.js

```diff
@@ -1,127 +1,103 @@
 const crypto = require('crypto')
-
 const _ = require('lodash')
-
 const dbs = require('./dbs')
 const env = require('./env')
-
-// error second API like a pro
 const ghBadgesCb = require('gh-badges')
-const ghBadges = (data) => new Promise((resolve, reject) => {
-  try {
-    ghBadgesCb(data, (result, err) => {
-      if (err) return reject(err)
-      resolve(result)
-    })
-  } catch (err) {
-    reject(err)
-  }
-})
 
-module.exports = async function badges (server, options, next) {
-  const enabled = await ghBadges({
-    format: 'svg',
-    template: 'flat',
-    colorscheme: 'brightgreen',
-    text: [
-      'Greenkeeper',
-      'enabled'
-    ]
-  })
-
-  const disabled = await ghBadges({
-    format: 'svg',
-    template: 'flat',
-    colorscheme: 'lightgray',
-    text: [
-      'Greenkeeper',
-      'disabled'
-    ]
+// error second API like a pro
+const ghBadges = data =>
```

Comment:
This is one of the candidates for why we prefer whitespace-only changes that don’t commit any functional changes:

 
![screen shot 2018-03-06 at 12 04 19](https://user-images.githubusercontent.com/11321/37029128-980e9b72-2136-11e8-9626-df0873b23a11.png)